external/vcpkg/ports/rocal/portfile.cmake

```diff
@@ -0,0 +1,39 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# cmake-format: off
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO ROCm/rocAL
+  REF 6efbed4807857b2b68a4ba90a043f8dd92f33c37
+  SHA512 9775088aa8e39efe10285d03c7050c16827bd3cf879e4b2cec61471e7660f7a55c22bd356060002e685dfe4de92fd476da4f9980a5afcd91dd8a7b9d39720df1
+  HEAD_REF develop
+)
+
+vcpkg_cmake_configure(
+  SOURCE_PATH ${SOURCE_PATH}
+  OPTIONS
+    -DBACKEND=CPU
+    -DBUILD_PYPACKAGE=OFF
+)
+# cmake-format: on
+
+vcpkg_cmake_install()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
```

Comment:
is it supposed to be debug here?

Reply:
Yes, this is fixed for inference server docker.